### PR TITLE
Added sample with configuration of physical interface with DHCP

### PR DESCRIPTION
**Pull request by 119Vik**
Added one more sample playbook to address issue #91

### CODE REVIEW

File: samples/ftd_configuration/physical_interface_with_default_dhcp.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            secondaryDNS: "{{ dhcp_containers_list.0.secondaryDNS }}"
+            primaryWINS: "{{ dhcp_containers_list.0.primaryWINS }}"
+            secondaryWINS: "{{ dhcp_containers_list.0.secondaryWINS }}"
+            servers: '{{ dhcp_containers_list.0.servers | selectattr("interface.name", "ne", iface_name) | list }}'
+            interface: "{{ dhcp_containers_list.0.interface }}"
+            type: "{{ dhcp_containers_list.0.type }}"
+        path_params:
+            objId: "{{ dhcp_containers_list.0.id }}"
+
+    - name: Setup Inside Interface with static IP
```
Comment (annikulin) [inline samples/ftd_configuration/physical_interface_with_default_dhcp.yml]: Do we need to update an interface? I guess nothing changes in its configuration, right?

File: samples/ftd_configuration/physical_interface_with_default_dhcp.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  vars:
+    iface_name: "inside"
+    iface_hardware_name: "GigabitEthernet0/1"
+  tasks:
+    - name: Get List of DHCP servers
+      ftd_configuration:
+        operation: "getDHCPServerContainerList"
+        register_as: dhcp_containers_list
+
+    - name: Execute 'editDHCPServerContainer' operation
```
Comment (annikulin) [inline samples/ftd_configuration/physical_interface_with_default_dhcp.yml]: Let's rename it to smth more specific, e.g., `Remove default DHCP server`.

File: samples/ftd_configuration/physical_interface_with_default_dhcp.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+- hosts: all
```
Comment (annikulin) [inline samples/ftd_configuration/physical_interface_with_default_dhcp.yml]: If I'm not wrong, the main part of this playbook is to remove default DHCP server. Does it make sense to rename it to smth like `dhcp_container_with_no_default_dhcp_server.yml`?

File: samples/ftd_configuration/physical_interface_with_default_dhcp.yml (hunk 1 above)
Comment (119Vik) [reply samples/ftd_configuration/physical_interface_with_default_dhcp.yml]: make sens. removed this step

File: samples/ftd_configuration/physical_interface_with_default_dhcp.yml (hunk 2 above)
Comment (119Vik) [reply samples/ftd_configuration/physical_interface_with_default_dhcp.yml]: renamed this step

File: samples/ftd_configuration/physical_interface_with_default_dhcp.yml (hunk 3 above)
Comment (119Vik) [reply samples/ftd_configuration/physical_interface_with_default_dhcp.yml]: makes sense as well.

**Pull request merged**